Dockerfile

```diff
@@ -11,7 +11,7 @@ RUN apt-get update && \
 RUN curl https://sh.rustup.rs -sSf | sh -s -- -y && \
          export PATH=$PATH:$HOME/.cargo/bin && \
          scripts/init.sh && \
-         cargo +nightly-2020-10-06 build --$PROFILE
+         cargo build --$PROFILE
```

Comment:
Which runtime version will be used here? It should be explicitly written.

Reply:
In init.sh, latest nightly will beused.